src/components/map/MapScreen.js

```diff
@@ -358,6 +358,7 @@ class MapScreen extends Component {
                     {this.state.city}, {this.state.region}
                 </Text>
                   <TouchableOpacity
+                    class="float-right"
```

Comment:
You need to use stylesheets and style = {stylesheet.name} bc class= doesnt work in rn.